### PR TITLE
CLI account unlock

### LINKED ISSUE #2348: Unlock geth account with password from CLI
Propose a solution that will allow us to achieve it without just passing password with plaintext that can be easily searched with comman line history.

### PR DESCRIPTION
**Pull request by maaktweluit**
Resolves #2348

### CODE REVIEW

File: golem/interface/client/account.py [hunk 1]
```diff
@@ -45,6 +47,33 @@ def info(self) -> Dict[str, Any]:
             )
         )
 
+    @staticmethod
+    @command(help="Unlock account, will prompt for your password")
+    def unlock() -> str:
+
```
Comment (Krigpl) [inline golem/interface/client/account.py]: rm ?

File: golem/interface/client/account.py [hunk 2]
```diff
@@ -45,6 +47,33 @@ def info(self) -> Dict[str, Any]:
...
+        if not has_key:
+            confirm = getpass.getpass('Confirm password:')
+            if confirm != pswd:
+                return "Password and confirmation do not match."
+            print("Generating keys, this can take up to 5 minutes...")
+        else:
+            print("Unlocking account...")
+
+        success = sync_wait(client.set_password(pswd), timeout=300)
```
Comment (Krigpl) [inline golem/interface/client/account.py]: The timeout should be higher, IIRC generating new key averages on 2 minutes, but p95 is 8minutes. And even recently @Elfoniok had to wait ~6minutes :)
This timeout is pretty harmless so might be like 10 minutes or more as well.

**Review by Krigpl — COMMENTED** [review]
lgtm, just a timeout thing

**Review by mfranciszkiewicz — APPROVED** [review]
LGTM

**Review by Wiezzel — COMMENTED** [review]
OK, but the timeout needs to be longer.

File: golem/interface/client/account.py (hunk 2 above)
Comment (maaktweluit) [reply golem/interface/client/account.py]: Changed it to 15 minutes timeout and the message to 10m, managing expectations here :)

**Review by Krigpl — APPROVED** [review]
verynicegreatsuccess

**Review by Wiezzel — APPROVED** [review]
OK

**Pull request merged**